aws/resource_aws_dax_cluster_test.go

```diff
@@ -0,0 +1,209 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dax"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsDaxCluster_basic(t *testing.T) {
+	var dc dax.Cluster
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsDaxClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsDaxClusterConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsDaxClusterExists("aws_dax_cluster.test", &dc),
+					resource.TestCheckResourceAttr(
```

Comment:
Can we please test the other attributes set by default? e.g. `port`, `maintenance_window`

Reply:
I've added more checks and altered some of the existing ones to match regexps